JS/popUp.js

```diff
@@ -0,0 +1,126 @@
+    image: 'imgMBV/Snapshoot 1.png',
+    technologies: ['html', 'css', 'javaScript'],
+    liveLink: 'https://ghostesso.github.io/Portfolio-New-Setup/',
+    sourceLink: 'https://github.com/GhostEsso/Portfolio-New-Setup',
+    details: [' CANOPY • UBER LEADER BOARD • 2015'],
+  },
+];
+// Créer une fonction qui va créer le HTML de chaque projet.
+for (let i = 0; i < projs.length; i += 1) {
+  workCont.innerHTML += `<section class="wo-section">
+      <div class="">
+        <img src="${projs[i].image}" alt="Snapshoot image">
+      </div>
+      <div class="">
+        <div class="w-topic">
+          <h2>${projs[i].title}</h2>
+        </div>
+        <div class="w-detail">
+          <ul>
+            <li>${projs[i].details}<li>
+          </ul>
+        </div>
+        <div class="w-paragraph">
+          <p>${projs[i].description}</p>
+        </div>
+        <div class="w-langs">
+          <ul>${projs[i].technologies.map((m) => `<li><a href="#">${m}</a></li>`).join('')}
+          </ul>
+        </div>
+        <div class="w-button">
+          <button id="btn-${[i]}" value="See Project">See Project</button>
+        </div>
+      </div>
+    </section>`;
+}
+// fenêtre pop-up
+for (let i = 0; i < projs.length; i += 1) {
+  const btnx = document.querySelectorAll(`[id=btn-${[i]}]`);
+  btnx.forEach((btx) => {
+    btx.addEventListener('click', () => {
+      container.style.display = 'flex';
+      container.innerHTML = `
+              <div class="w-m-c pop container-fluid p-5">
+                <div class="row bg-light work-re">
+                  <span class="closeit"><i class="fa-solid fa-xmark"></i></span>
+                  <div class="col-12">
+                    <div class="w-m-h px-2 py-2">
+                      <h1 class="topic-here">${projs[i].title}</h1>
+                      <ul class="list-inline flex">
+                        ${projs[i].details}
+                      </ul>
+                      <img src="${projs[i].image}" class="img-fluid w-100" alt="Logo">
+                    </div>
+                  </div>
+                  <div class="col-12">
+                    <div class="row d-flex flex-row">
+                      <div class="col-md-8">
+                        <p class="paragraph pb-2">${projs[i].longDescription}</p> 
+                      </div>
+                      <div class="col-md-4 d-flex flex-column">
+                        <ul class="list-inline ul-work">
+                          ${projs[i].technologies.map((m) => `<li class="list-inline-item bagde bg-primary text-primary"><a class="text-decoration-none text-dark" href="#">${m}</a></li>`).join('')}
+                        </ul>
```

Comment:
Nice work mapping through the list of technologies 💯 but the style needs some improvement. Take a look at the differences 👇🏽 

## Figma
![Screenshot 2023-03-16 at 09 12 17](https://user-images.githubusercontent.com/45739963/225627466-18d70648-902c-499f-96ca-b5ae071f6e95.png)

## Your design
![Screenshot 2023-03-16 at 09 11 52](https://user-images.githubusercontent.com/45739963/225627347-cfe29881-b0c3-4737-9719-4715e21952ff.png)

The spacing, and colors, are not quite the same and it also lacks that separator between the technologies and the buttons. 🔬